services/webhook/tests/test_complexity.py

```
@@ -0,0 +1,99 @@
+            "def f(x):\n"
+            "    if x:\n"
+            "        if x:\n"
+            "            if x:\n"
+            "                pass\n"
+        )
+        # 3 flat ifs = 1+1+1 = 3; nested = 1 + 2 + 3 = 6 (deeper reads worse)
+        assert cognitive_complexity(flat) == 3
+        assert cognitive_complexity(nested) == 6
+
+
+def _hunk(path, start, added):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_hunk`

(ANN202)